xs2a-adapter-rest-impl/src/main/resources/static/main.js

```diff
@@ -455,4 +440,40 @@ function addTooltips(e) {
         helper.setAttribute("class", "mdl-tooltip mdl-tooltip--top");
     }
 }
+function persist() {
+    fetch("v1/aspsps/persist", {
+        method: "POST"
+    })
+        .then(response => {
+            if (!response.ok) {
+                throw Error(response.statusText);
+            }
+            success();
+        })
+        .catch(error => {
+            console.log(error);
+            operationFailed();
+        })
+}
+
+function paginate(data) {
+    let dataLength = data.length;
+    let step = 10;
+    let current = 0;
+    let button = document.querySelector(".show-more");
+    let total = document.querySelector(".total");
+
+    addPage();
+    total.innerHTML = dataLength;
+
+    function addPage() {
+        for (limit = current + (step > dataLength ? dataLength : step); current < limit; current++) {
+            buildRow(data[current]);
+        }
+        current < dataLength ? button.hidden = false : button.hidden = true;
```

Comment:
It seems to me that this line of code could be simplified to smth like:
```
button.hidden = current >= dataLength;
```
What do you think?

Reply:
hmm... good approach, thx